### ISSUE TITLE
Started ML Book (08/23)

### ISSUE THREAD

**Issue by anitabe404** [studydash/cards#336]
## Your Daily Accomplishment 🚀:
Started reading Hands-on Machine Learning with Scikit-Learn, Keras, & TensorFlow by Aurelien Geron.
Started a PyQt tutorial on creating GUIs. (Still exploring options for my project's GUI)
Reviewed Python OOP concepts on Codecademy.

## Extra/Fun (Optional) 🎈🎉🥳:
No fun for me today. LOL

**Comment by r002** [comment]
> _No fun for me today. LOL_

@anitabe404 Oh no!!  Fun is a central ingredient!  Working hard is good, but gotta also enjoy life once in a while! You got this! 😄

**Comment by anitabe404** [comment]
@r002 Yes, I've been working on this. I picked up a new book. It's pretty cool.

[Requiem Moon by C.T. Rwizi](https://www.amazon.com/Requiem-Moon-Scarlet-Odyssey-Book-ebook/dp/B087Q18QT4) It's book 2 of a duology. I read the first book last year.
